server/app/src/androidTest/java/sh/calaba/instrumentationbackend/StatusReporterObject.java

```diff
@@ -0,0 +1,76 @@
+package sh.calaba.instrumentationbackend;
+
+import android.os.Environment;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class StatusReporterObject {
+
+    public static void report(String method,
+                              String message,
+                              StatusReporter.FinishedState extraState,
+                              boolean hasData) {
+
+
```

Comment:
Nitpick: too many whitespaces in this method

Reply:
true